config/optional/core.entity_form_display.paragraph.video.default.yml

```diff
@@ -16,13 +16,14 @@ content:
     weight: 0
     settings:
       entity_browser: video_browser
-      field_widget_display: inline_entity_form
+      field_widget_display: rendered_entity
+      field_widget_display_settings:
+        view_mode: thumbnail
```

Comment:
On open paragraphs, the image and the video preview/thumbnail have different sizes